ci/benchmarks/partial-conv/evo2_pretrain.yaml

```diff
@@ -1,5 +1,5 @@
-scope: partial-conv
-time_limit: 14400
+scope: perf
+time_limit: 900
```

Comment:
why is the "scope: perf" in the partial-conv dir?

Reply:
thank you! great catch! fixed